hack/scripts/aws/setup_aws_accounts.sh

```diff
@@ -0,0 +1,146 @@
+
+    check_var "AWS_SESSION_TOKEN" "$AWS_SESSION_TOKEN"
+    export AWS_SESSION_TOKEN
+}
+
+#########################
+# Account 1 - Jump Role
+#########################
+assume_aws_role ${AWS_ACCOUNT_ID_1} osd-staging-1
+
+# Create the sts Assume role Policy which will be applied to the new Role
+ASSUME_ROLE_POLICY_ARN=$(aws iam create-policy --policy-name STSAssumeRolePolicy${ID} --policy-document file://./setup-aws-policies/AssumeRolePolicy.json | jq '.Policy.Arn')
+# Strip quotes from policy arn
+ASSUME_ROLE_POLICY_ARN=$(echo ${ASSUME_ROLE_POLICY_ARN} | tr -d '"')
+
+### Create the Jump Role
+TRUST_ARN_FORMAT="arn:aws:iam::277304166082:user/"${USERNAME}
+# Here we're updating the ARN value set in the json file with the appropriate value
+PREPARED_JSON=$(cat setup-aws-policies/JumpRoleTrustRelationship.json | jq --arg TRUST_ARN_FORMAT $TRUST_ARN_FORMAT '.Statement[0].Principal.AWS |= $TRUST_ARN_FORMAT')
+# Create the Role and save its ARN, we'll use it later.
+JUMP_ROLE_ARN=$(echo $PREPARED_JSON | aws iam create-role --role-name JumpRole${ID} --assume-role-policy-document file:///dev/stdin | jq '.Role.Arn')
+# Attach the ARN generated by the Policy to the Jump Role we created
+ATTACH_ROLE_POLICY=$(aws iam attach-role-policy --role-name JumpRole${ID} --policy-arn "${ASSUME_ROLE_POLICY_ARN}")
+
+
+#########################
+# Account 2 - Access Role
+#########################
+assume_aws_role ${AWS_ACCOUNT_ID_2} osd-staging-2
+
+# Stript quotes from jump ARN
+JUMP_ROLE_ARN=$(echo ${JUMP_ROLE_ARN} | tr -d '"')
+# Update AccessRoleTrustRelationship with the following Jump Role ARN
+PREPARED_JSON=$(cat setup-aws-policies/AccessRoleTrustRelationship.json | jq --arg JUMP_ROLE_ARN ${JUMP_ROLE_ARN} '.Statement[0].Principal.AWS |= $JUMP_ROLE_ARN')
+
+# Here we need to add a backoff retry as we may encounter the following error from aws:
+## An error occurred (MalformedPolicyDocument) when calling the CreateRole operation: Invalid principal in policy: "AWS":"arn:aws:iam::190496588311:role/JumpRole"
+# AWS is running checks to ensure that the principal exists. Because of IAM's eventual consistency model, 
+# even though the first command to create the Jump Role works and returns an ARN, it takes time to fully 
+# propagate through AWS systems. Therefore this command to create the Access Role can fail a few times 
+# before the Role ARN returns as "valid".
+
+max_retries=5
+i=1
+while ! STS_ROLE_ARN=$(echo $PREPARED_JSON | aws iam create-role --role-name AccessRole${ID} --assume-role-policy-document file:///dev/stdin)
+do
+  if [[ $i > $max_retries ]]
+  then
+    echo "Could not create jump role"
+    exit 1
+  fi
+  ((j=2**i))
+  echo "Jump Role Arn not ready yet."
+  echo "sleeping $j"
+  sleep $j
+  ((i=i+1))
+done
+
+STS_ROLE_ARN=$(echo ${STS_ROLE_ARN} | jq '.Role.Arn')
+
+echo "Your Accounts have been setup successfully!"
+echo "Place the following in your env variables for the AAO"
+echo "STS_ROLE_ARN="${STS_ROLE_ARN}
```

Comment:
The quoting on these lines is strange.  Can we adjust this to be more standard? Unless there's a need for the quoting like this, in which case we should add a comment as to why the quoting ends before the variable.